earn/src/pages/BoostPage.tsx

```diff
@@ -227,14 +254,16 @@ export default function BoostPage() {
           );
         })}
       </div>
-      <ImportModal
-        isOpen={selectedPosition !== null && selectedCardInfo?.cardType === BoostCardType.UNISWAP_NFT}
-        uniqueId={selectedCardInfo ? getUniqueId(selectedCardInfo) : ''}
-        setIsOpen={() => {
-          setSelectedPosition(null);
-        }}
-        cardInfo={selectedCardInfo}
-      />
+      {selectedCardInfo !== undefined && (
```

Comment:
Why is this necessary for the ImportModal but not the ManageBoostModal?

Reply:
I haven't updated the `ManageBoostModal` yet. It just allows us to not have undefined `cardInfo` in the modal.